packages/domain-picker/src/domain-picker/style.scss

```diff
@@ -449,3 +454,17 @@ $accent-blue: #117ac9;
 		color: var( --wp-admin-theme-color-darker-10 );
 	}
 }
+
+.domain-picker__action-link {
+	display: flex;
+	align-items: center;
+
+	&.with-underline {
+		text-decoration: underline;
+	}
+
+	a,
+	a:hover {
+		color: var(--mainColor);
+	}
+}
```

Comment:
I'm looking at this `__action_link` and at the `GoBackButton` in the `@automattic/launch` package and they seem to be styled the same in Figma. Maybe it makes sense for them to share the same CSS and markup (with a prop `arrow: 'left' | 'right'`) and be exported from the `@automattic/onboarding` package?

It's also fine to leave that extraction for later (add a comment in code linking to this comment) and just [copy the CSS](https://github.com/Automattic/wp-calypso/blob/trunk/packages/launch/src/focused-launch/go-back-button/style.scss) to set the correct hover opacity, background, border, and other resets.

Reply:
I've moved this to a component and created an issue to replace the Back Buttons with the newly created component https://github.com/Automattic/wp-calypso/issues/48568